tendrl/monitoring_integration/upgrades/utils.py

```diff
@@ -0,0 +1,40 @@
+import grp
+import os
+import pwd
+
+
+def stop_service(service):
+    os.system("systemctl stop %s" % service)
+
+
+def start_service(service):
+    os.system("systemctl start %s" % service)
+
+
+def command_exec(cmd):
+    os.system(cmd)
+
+
+def remove_file(path):
+    try:
+        os.remove("/var/lib/graphite-web/graphite.db")
+    except OSError as ex:
+        print (ex)
+
+
+def change_owner(path, owner, recursive=False):
```

Comment:
It this function should be "universally" used, it would be probably better, to separate *owner* to *user* and *group* arguments (instead of using one value for both.

Reply:
ah, i will modify and add exception handler for this